test/e2e-tests.sh

```diff
@@ -22,10 +22,21 @@ apply_chaos || fail_test "Failed to apply chaos"
 
 header "Waiting Knative eventing to come up"
 
-wait_until_pods_running knative-eventing || fail_test "Knative Eventing did not come up"
+apply_sacura || fail_test "Failed to apply Sacura"
```

Comment:
Can we have a flag to enable/disable this thing when we run this script?

Reply:
Sure!